userspace/qtwayland/README

```diff
@@ -0,0 +1,79 @@
+# Ubuntu 24.04 + "old" weston
+
+The following need to work together in Ubuntu 24.04 (noble):
+- "old" weston
+- libwayland
+- qtwayland
+- qt
+
+The "old" weston is built with:
+- libwayland-dev_1.9.0-1
+- libwayland-server_1.9.0-1
+- libwayland-client_1.9.0-1
+- libwayland-cursor_1.9.0-1
+
+These are roughly from Xenial (16.04 - https://launchpad.net/ubuntu/xenial/arm64/libwayland-dev/1.9.0-1).
+
+In Ubuntu 20.04 (focal) even if libwayland is 1.18.0-1, the protocol between
+the server (1.9.0-1) and client (1.18.0-1) was still compatible. Unfortunately,
+this is not the case anymore in Ubuntu 24.04 (noble), where libwayland 1.22.0
+is not compatible anymore with 1.9.0.
+
+Because of this:
+- libwayland-*_1.9.0-1 packages can't be installed anymore from noble, but a
+custom deb has been built manually (libwayland-1.9.0-1.deb) - this includes all
+libwayland packages in one package - there is no source for this deb
+- libwayland-*_1.9.0-1 packages has a dependency (libffi6)
+
+Also, Qt 5.15 from Ubuntu 24.04 (noble) doesn't compile with these old
+dependencies, 5.12 being the last major being able to be compiled, so:
+- a custom qt-5.12.8.deb has been built manually, since qt from focal has a lot
+of dependencies not existing in Ubuntu 24.04 (noble) - there is no source for
+this deb
+- qt has two non-qt related dependencies (libicu66, libssl1.1)
```

Comment:
can you provide commands to build these? any binary that's checked in needs instructions